executor/executor.cc

```diff
@@ -428,6 +432,7 @@ int main(int argc, char** argv)
 			// Don't enable comps because we don't use them in the fuzzer yet.
 			cover_enable(&extra_cov, false, true);
 		}
+		init_bitmap();
```

Comment:
> coverage_filter is called only if flag_coverage_filter is set and it fails if bitmap is not initialized, while init_bitmap uses completely different pattern - it's called always and does not fail if the file is not present. Call init_bitmap only if flag_coverage_filter is set and fail if the file is not present for consistency.
I had met some difficulty before.
Since coverage_filter will cause coverage check to fail ( checkMachine), if init_coverage_filter called only when flag_coverage_filter is enabled, it should be moved to somewhere can be executed when a new prog( its execopt) is received.
So the coverage check would not be failed, but init_coverage_filter will be called frequently( add a check to avoid reinitializing an initialized bitmap).
Maybe it should be moved to execute_one? Do you think it's appropriate?

Reply:
I see. I missed that we will receive the flag only later.
I think your current code is good them. Just add this comment to avoid future confusion:
```
	int f = open("/syz-cover-bitmap", O_RDONLY);
	if (f < 0) {
		// We don't fail here because we don't know yet if we should use coverage filter or not.
		// We will receive the flag only in execute flags and will fail in coverage_filter if necessary.
		debug("bitmap is no found, coverage filter disabled\n");
		return;
	}
```

Reply:
> coverage_filter and init_bitmap are used and named inconsistently.
> Let's call everything related to coverage filtering as "coverage_filter", e.g. init_bitmap should then be init_coverage_filter.

To keep all the code about coverage filter consistent.
I renamed:
1. executor/bitmap.h to executor/cov_filter.h.
2. syz-manager/filter to syz-manager/covfilter.go.
3. Rename fields of structure CoverFilter( covfilter.go) as cov_filter_t( cov_filter.h).